### PR TITLE
go: GORP framework support

### PR DESCRIPTION
**Pull request by hiteshbedre**

### CODE REVIEW

File: src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    fileNode: NodeOrDetachedNode
+  ): Unit = {
+    try {
+      val sqlTable: SQLTable = SQLTable(
+        model.name,
+        model.lineNumber.getOrElse(Integer.valueOf(-1)),
+        model.columnNumber.getOrElse(Integer.valueOf(-1))
+      )
+      val sqlColumns: List[SQLColumn] = model.member.l.map(x =>
+        SQLColumn(x.code, x.lineNumber.getOrElse(Integer.valueOf(-1)), x.columnNumber.getOrElse(Integer.valueOf(-1)))
```
Comment (dbMundada) [inline src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala:66]: rename `x` to `member`

File: src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        fileNode,
+        queryModel,
+        model.code,
+        model.lineNumber.getOrElse(Integer.valueOf(-1)),
+        0
+      )
+    } catch {
+      case ex: Exception =>
+        ex.printStackTrace()
+        println(s"Error while building SQL nodes: ${ex.getMessage}")
```
Comment (dbMundada) [inline src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala:80]: Use standard logger

File: src/main/scala/ai/privado/languageEngine/go/passes/orm/GorpParser.scala [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ *
+ */
+package ai.privado.languageEngine.go.passes.orm
+
+import ai.privado.cache.RuleCache
```
Comment (dbMundada) [inline src/main/scala/ai/privado/languageEngine/go/passes/orm/GorpParser.scala:25]: Can we remove unwanted imports

Comment (hiteshbedre) [reply src/main/scala/ai/privado/languageEngine/go/passes/orm/GorpParser.scala]: Addressed here: 3bc98acb

File: src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala (hunk 1 above)
Comment (hiteshbedre) [reply src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala]: Addressed here: 3bc98acb

File: src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala (hunk 2 above)
Comment (hiteshbedre) [reply src/main/scala/ai/privado/languageEngine/go/passes/orm/BaseORMParser.scala]: Addressed here: 6377ce46

**Review by dbMundada — APPROVED** [review]
LGTM

**Pull request merged**